packages/grafana-e2e-selectors/src/selectors/pages.ts

```diff
@@ -122,5 +127,8 @@ export const Pages = {
       container: 'Explore',
       runButton: 'Run button',
     },
+    Toolbar: {
+      navBar: '.explore-toolbar',
+    },
```

Comment:
This looks weird, not used anywhere in the updated code?

Reply:
It's used in code that will be in my next PR. Including it here makes it easier to test against the nightly-master server,